src/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.cs

```diff
@@ -44,6 +44,11 @@ public sealed partial class Collection : System.Collections.ICollection, System.
         public void Remove(int Index) { }
         public void Remove(string Key) { }
     }
+    public enum CompareMethod
+    {
+        Binary = 0,
+        Text = 1,
+    }
```

Comment:
Please move `CompareMethod` back to original location.

Reply:
It is where it was before I started in Commit 2dde65